chrome/skin/style-linux.css

```diff
@@ -0,0 +1,4 @@
+#universal-search-recommendation {
```

Comment:
Since the platform-specific styles are just for highlights, you want this selector to be `#universal-search-recommendation.highlight`.